profiles/kentik_snmp/cisco/cisco-firepower.yml

```diff
@@ -1,13 +1,47 @@
-# https://www.cisco.com/c/en/us/products/collateral/security/firepower-ngfw/white-paper-c11-741739.html#_Toc5440015
-# This profile uses the UC Davis MIB for CPU and Memory based on Cisco's doc above
+# http://www.circitor.fr/Mibs/Html/C/CISCO-ENHANCED-MEMPOOL-MIB.php
 ---
 extends:
   - if-mib.yml
   - system-mib.yml
-  - ucd-mib.yml
 
 provider: kentik-firepower
```

Comment:
Do we want to create a whole new entity for this or should it just be a firewall?

Reply:
i'm going to add it as a firewall; but leaving this provider value in case it makes sense later to split it out by itself. that way we don't make a breaking change if it comes to it